src/main/java/com/geeksforless/client/model/entity/CustomUserDetails.java

```diff
@@ -0,0 +1,55 @@
+package com.geeksforless.client.model.entity;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+
+public class CustomUserDetails implements UserDetails {
+    private Long id;
+    private String email;
+    private String password;
+
+    public static CustomUserDetails fromUserToCustomUserDetails(User user) {
+        CustomUserDetails userDetails = new CustomUserDetails();
+        userDetails.id = user.getId();
+        userDetails.email = user.getEmail();
+        userDetails.password = user.getPassword();
+        return userDetails;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
```

Comment:
Are you sure that in this place must be null?

Reply:
because we dont use roles and permissions now, this will be null